contrib/Trivy.gitlab-ci.yml

```diff
@@ -0,0 +1,36 @@
+Trivy_container_scanning:
+  stage: test
+  image:
+    name: alpine
+  variables:
+    # Override the GIT_STRATEGY variable in your `.gitlab-ci.yml` file and set it to `fetch` if you want to provide a `clair-whitelist.yml`
+    # file. See https://docs.gitlab.com/ee/user/application_security/container_scanning/index.html#overriding-the-container-scanning-template
+    # for details
+    GIT_STRATEGY: none
+    IMAGE: "$CI_REGISTRY_IMAGE:$CI_COMMIT_SHA"
+  allow_failure: true
+  before_script:
+    - apk add --no-cache curl docker-cli
+    - docker login -u "$CI_REGISTRY_USER" -p "$CI_REGISTRY_PASSWORD" $CI_REGISTRY
+    - export VERSION=$(curl --silent "https://api.github.com/repos/aquasecurity/trivy/releases/latest" | grep '"tag_name":' | sed -E 's/.*"v([^"]+)".*/\1/')
+    - echo $VERSION
+    - wget https://github.com/aquasecurity/trivy/releases/download/v${VERSION}/trivy_${VERSION}_Linux-64bit.tar.gz
+    - tar zxvf trivy_${VERSION}_Linux-64bit.tar.gz
```

Comment:
Would you replace these lines with an install script?
https://github.com/aquasecurity/trivy-ci-test/blob/bc965f9a1939ab932e6882e64bad990cffbc17d2/.circleci/config.yml#L15

Reply:
I tested it here.
https://gitlab.com/knqyf263/trivy-ci-test/commit/9338315bb03dace4a14c5d4b97a4e40011ac9866#587d266bb27a4dc3022bbed44dfa19849df3044c_0_32

Reply:
Nice improvement! Fix it in d4e0cc9.